dss-core/src/main/java/io/github/ztkmkoo/dss/core/actor/rest/DssRestAuthActor.java

```diff
@@ -0,0 +1,131 @@
+package io.github.ztkmkoo.dss.core.actor.rest;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.crypto.SecretKey;
+
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthCommand;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthenticationCommandRequest;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthenticationCommandResponse;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthorizationCommandRequest;
+import io.github.ztkmkoo.dss.core.message.security.DssAuthorizationCommandResponse;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import io.jsonwebtoken.JwtException;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.security.Keys;
+
+public class DssRestAuthActor {
+	
+	public static Behavior<DssAuthCommand> create(Map<String, String> userList) {
+        return Behaviors.setup(context -> new DssRestAuthActor(context, userList).dssRestAuthActor());
+    }
+	
+	private final ActorContext<DssAuthCommand> context;
+	private final Map<String, String> userList;
+	private final Map<String, SecretKey> tokenList;
+    
+    private DssRestAuthActor(ActorContext<DssAuthCommand> context, Map<String, String> userList) {
+        this.context = context;
+        this.userList = userList;
+        this.tokenList = new HashMap<>();
+    }
+    
+    private Behavior<DssAuthCommand> dssRestAuthActor() {
+        return Behaviors
+                .receive(DssAuthCommand.class)
+                .onMessage(DssAuthenticationCommandRequest.class, this::handlingDssAuthenticationCommandRequest)
+                .onMessage(DssAuthorizationCommandRequest.class, this::handlingDssAuthorizationCommandRequest)
+                .build();
+    }
+    
+    private Behavior<DssAuthCommand> handlingDssAuthenticationCommandRequest(DssAuthenticationCommandRequest request) {
+    	
+    	context.getLog().info("DssAuthenticationCommandRequest: {}", request);
+    	
+    	String userID = request.getUserID();
+    	String userPassword = request.getUserPassword();
+    	String token = null;
+    	SecretKey key = Keys.secretKeyFor(SignatureAlgorithm.HS256);
```

Comment:
Do we need create new secret key for every token?
Can we make just one key, and sign the jwt token?

I think it works because we put the payload with user id when create the token.

Reply:
So you think just make one key in actor? I'll try it.

Reply:
Yap. I think one key is enough.

Reply:
And do not send key back.

Reply:
In payload, there is userID already, so don't need to make new secret key. Agree with your review.
It makes codes more simple. Thanks!